bin/brew

```diff
@@ -72,7 +73,7 @@ then
 
   FILTERED_ENV=()
   # Filter all but the specific variables.
-  for VAR in HOME SHELL PATH TERM COLUMNS LOGNAME USER CI TRAVIS SSH_AUTH_SOCK SUDO_ASKPASS \
+  for VAR in HOME SHELL PATH TEMP TERM COLUMNS LOGNAME USER CI TRAVIS SSH_AUTH_SOCK SUDO_ASKPASS \
```

Comment:
Why do we want to add this?

Reply:
Oh, I misread this code. I thought this was explicitly whitelisting `HOMEBREW_TEMP`, but I guess it would clobber it instead.